src/map/status.cpp

```diff
@@ -13331,7 +13327,7 @@ int32 status_change_end( struct block_list* bl, enum sc_type type, int32 tid ){
 			unit_data *ud = unit_bl2ud(bl);
 
 			if (ud)
```

Comment:
```suggestion
			if (ud != nullptr) {
```

Reply:
Here you touched the line.

Reply:
I didn't. It's white.

Reply:
Ah yeah I suggested it here, because I had to suggest a { for the changes below.